docs/known_generator_issues.md

```diff
@@ -0,0 +1,82 @@
+# Known issues
```

Comment:
If this is going to be checked in while we still have multiple backends, this document should indicate which backend these known issues are associated with.

Reply:
Nope. After Viktor's fix.

Reply:
So, we'd have just Roslyn backend